src/features/profile/pages/NotificationSettings/NotificationsSettings.tsx

```diff
@@ -102,7 +102,7 @@ export const NotificationsSettings = () => {
 
   const togglePush = () => {
     if (pushPermission === 'granted') {
-      dispatch({ type: 'push' })
+      dispatch({ type: 'push', state: 'toggle' })
```

Comment:
Au lieu d'avoir `'toggle' | boolean` comme type, est-ce qu'on peut pas avoir que des booléans ?
```suggestion
      dispatch({ type: 'push', state: !state.allowPush })
```

Reply:
Non justement, c'est ce qu'on avait avant et ça me faisait des comportements hyper bizarres que je n'ai pas forcément compris mais j'ai besoin de forcer un état à true et pas juste un toggle (je peux t'envoyer une vidéo du comportement actuel avec ça c'est hyper bizarre)

Reply:
Je comprend pour le besoin de forcer l'état à true, mais je ne comprends pas l'utilisation de `toggle` au lieu de `!state.allowPush`, on peut en parler demain